Update 4.6 & 3.3 PHP requirements (#3234)
ezsystems/developer-documentation: docs/update_and_migration/from_4.6/update_from_4.6.md

#### Update PHP, the custom code, then the platform (recommended)

> Make sure to use PHP 8.1 minimum. Notice that PHP 8.1 has reached its End of Life so you should consider using PHP 8.2 or higher.

Reviewer: ```suggestion
Make sure to use PHP 8.1 or higher. Since PHP 8.1 has reached its End of Life (EOL), it's recommended that you use PHP 8.2 or higher.
```
